rust/headless-client/src/lib.rs

```diff
@@ -50,7 +50,7 @@ struct Cli {
     // TODO: It isn't good for security to pass the token as a CLI arg.
     // If we pass it as an env var, we should remove it immediately so that
     // other processes don't see it. Reading it from a file is probably safest.
-    #[arg(env = "FIREZONE_TOKEN")]
+    #[arg(long, env = "FIREZONE_TOKEN")]
```

Comment:
This breaks CLI compatibility but having the token as a positional arg was messing up the compatibility tests because `firezone-headless-client looks_like_a_token_on_old_versions_or_a_subcommand_on_new_versions` does what it says